cmd/new.go

```diff
@@ -234,7 +241,29 @@ func newNewCmd() *cobra.Command {
 				fmt.Println("Finished installing dependencies.")
 
 				// Write a summary with next steps.
-				fmt.Println("New project is configured and ready to deploy with 'pulumi update'.")
+				fmt.Println("New project is configured and ready to deploy.")
+
+				// If the current working directory changed, add instructions to
+				// cd into the directory.
+				if originalCwd != cwd {
+					// If we can determine a relative path, use that, otherwise use
+					// the full path.
+					var cd string
+					if rel, err := filepath.Rel(originalCwd, cwd); err == nil {
+						cd = rel
+					} else {
+						cd = cwd
+					}
+
+					// Surround the path with double quotes if it contains whitespace.
+					if containsWhiteSpace(cd) {
+						cd = fmt.Sprintf("\"%s\"", cd)
+					}
+
+					fmt.Printf("Run 'cd %s' then 'pulumi update'.\n", cd)
```

Comment:
On purpose that there is a trailing newline here but not in the other branch?

Reply:
The `else` uses `Println` whereas `Printf` with explicit `\n` is used here. (I really wish go had a built-in `Printlnf`).

Reply:
Ahh - yeah - totally missed that.  

I just completely avoid `Println` myself to avoid this confusion :smile:.